sdk/react/widgets/FwdButton.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {
+  TouchableHighlight,
+  Animated,
+  View,
+  Text
+} from 'react-native';
+
+const AccessibilityUtils = require('../accessibilityUtils');
+const Constants = require('../constants');
+const styles = require('../utils').getStyles(require('./style/RectButtonStyles.json'));
+
+class FwdButton extends React.Component {
```

Comment:
instead of FwdButton can we change it to SkipButton. SkipButton seems more relevant with the feature.

Reply:
@akshaydch11 Renamed